AugmentedNet/annotation_parser.py

```diff
@@ -131,10 +131,12 @@ def _extractRomanNumeralInformation(rn):
         and "Fr" not in rn.figure
         and "It" not in rn.figure
     ):
-        if rn.figure in ["I4", "iv4", "v4", "V4"]:
```

Comment:
See the discussion at https://github.com/cuthbertLab/music21/issues/1218

Reply:
Thank you for bringing this up! I have no idea how to deal with this, to be honest.